runtime/pom.xml

```diff
@@ -21,27 +21,37 @@
         </dependency>
         <dependency>
             <groupId>org.apache.tika</groupId>
-            <artifactId>tika-parsers</artifactId>
+            <artifactId>tika-core</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.poi</groupId>
-            <artifactId>ooxml-schemas</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>jakarta.annotation</groupId>
-            <artifactId>jakarta.annotation-api</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.jboss.spec.javax.xml.bind</groupId>
-            <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.jboss.slf4j</groupId>
-            <artifactId>slf4j-jboss-logmanager</artifactId>
+            <groupId>org.apache.tika</groupId>
+            <artifactId>tika-parsers-standard-package</artifactId>
```

Comment:
The problem with this dependency is that it is a shaded JAR that resolves all the other modules as transitive dependencies too. I created https://github.com/apache/tika/pull/936 as an attempt to fix that

Reply:
Seems reasonable, my first try was to add a lot of exclusions for those transitive dependencies, but there was always something new breaking... This seems a lot cleaner approach.

Reply:
I think it'd be better if we include each transitive dependency from this artifact directly. That would give us a finer control over the dependencies

Reply:
I’ve managed to get to 72 parser when including individual parser modules instead of whole standard package. Will have to check where are the few missing ones coming from…
This could be a next step, but first I wanted to verify the tests in native mode with quarkus-poi.

Reply:
Missing parsers:
org.apache.tika.parser.external.CompositeExternalParser
org.gagravarr.tika.SpeexParser
org.gagravarr.tika.FlacParser
org.gagravarr.tika.VorbisParser
org.gagravarr.tika.TheoraParser
org.gagravarr.tika.OggParser
org.gagravarr.tika.OpusParser

Looks like this is another external dependency (defined in standard parsers pom), if all this parsers should be supported out of the box.